client/src/pages/Feed.js

```diff
@@ -258,6 +260,41 @@ const Feed = () => {
     dispatchAction(TOGGLE_STATE, "showFilters");
   }
 
+  const handlePostLike = (postId) => {
+    const { history, isAuthenticated, user } = props;
+
+    /* added here because userId not working */
+    sessionStorage.removeItem("likePost");
+
+    if (isAuthenticated) {
+      const endPoint = `/api/posts/${postId}/likes/${user && user.userId}`;
+      const post = posts.posts.filter((post) => post._id === postId);
+
+      if (user) {
+        if (!post[0].liked) {
+          axios.put(endPoint)
+            .then((response) => {
+              postsDispatch({ type: SET_LIKE, postId, count: response.data.likesCount });
+            })
+            .catch(error => {
```

Comment:
Should we store the errors in Redux? That way the Post can access errors and display them. What do you think?